migrations/1659106715328-MaterializedView.ts

```diff
@@ -0,0 +1,48 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class MaterializedView1659106715328 implements MigrationInterface {
+  name = "MaterializedView1659106715328";
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`drop materialized view if exists deposits_mv`);
+    await queryRunner.query(`
+      create materialized view deposits_mv as
+        select
+          d."depositId",
+          d."depositTxHash",
+          d."sourceChainId",
+          d."destinationChainId",
+          d.amount,
+          t.symbol,
+          t.decimals,
+          d."depositorAddr",
+          d."stickyReferralAddress" as "referralAddress",
```

Comment:
Use the `stickyReferralAddress` column instead of `referralAddress`. `referralAddress` stores only the on-chain referral address